p5/index.qmd

```diff
@@ -4,7 +4,291 @@ bibliography: ../tds.bib
 toc: true
 execute: 
   cache: true
+  eval: true
+editor: 
+  markdown: 
+    wrap: 72
 ---
-This is a placeholder page; content will be added soon.
 
+# Introduction
 
+In this practical, we will build on the **routing techniques** from Practical 4 by exploring **data visualization** methods for transport analysis. By the end of this practical, you should be able to:
+
+- **Process** real transport demand datasets  
+- **Create** flow maps and isochrones  
+- **Analyze** network centrality  
+- **Visualize** multi-modal journeys  
+- **Generate** 3D transport visualizations  
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  message = FALSE,
+  warning = FALSE,
+  fig.width = 7,
+  fig.height = 5,
+  out.width = "700px"
+)
+```
+
+# Setup
+Below are the libraries we will use throughout this practical:
+```{r}
+# Load necessary libraries
```

Comment:
I suggest removing some of these to the minimum to fix the failures on GitHub Actions.